doc/api/dns.md

```diff
@@ -184,7 +184,8 @@ changes:
     addresses in the order the DNS resolver returned them. When `false`,
     IPv4 addresses are placed before IPv6 addresses.
     **Default:** currently `false` (addresses are reordered) but this is
-    expected to change in the not too distant future.
+    expected to change in the not too distant future. Default value is
+    configurable using [`dns.verbatim`][] or [`--dns-order`][].
```

Comment:
The doc in cli.md implies that verbatim is the default

Reply:
Description changed.